CHANGELOG.md

```diff
@@ -49,6 +147,28 @@
   on the network).
   (#2412 @shaneutt)
 
+### Documentation
+- A guide for Gateway API implementers is now included in the specification.
+  (#2454, @youngnick)
+- Gateway API versioning will continue to rely on two release channels -
+  Standard and Experimental. New resources will start in the Experimental
+  Channel with an alpha API Version and then graduate to the Standard Channel
+  with a GA API version. Resources that already have Beta API versions will
+  continue to have them, but no additional Gateway API resources will get a Beta
+  API version. (#2446, @robscott)
+
+### Cleanup
+- Validating Webhook logs now use `Errorf` instead of `Fatalf`. (#2361, @yylt)
+
+### Other (Cleanup or Flake)
```

Comment:
not a blocker persay, but should we draw extra attention to this item?

Reply:
Yep, good call, added new section above, thanks!